usaspending_api/references/management/commands/load_reference_data.py

```diff
@@ -19,7 +19,7 @@ def handle(self, *args, **options):
         call_command("loaddata", "reference_fixture")
 
         self.logger.info("Loading agency list")
-        call_command("load_agencies")
+        call_command("load_agencies", "https://files.usaspending.gov/reference_data/agency_codes.csv")
```

Comment:
If we're hardcoding the URL, why not just commit it to code?

Reply:
You're preaching to the choir.  I received pushback when I suggested it and didn't feel passionate enough about the issue to press it.  Technically, the file is owned by Broker so I guess it's their decision to make.

Reply:
Can we put it in the readme then? This is an un-obvious place to find the latest agency list if you bork the table and just want to reload agencies.

Reply:
settings.py is a good location to store the URL. Other URLs are stored there already and they handle the prod-vs-nonprod differences